addons/web_editor/static/lib/odoo-editor/test/spec/copyPaste.test.js

```diff
@@ -348,34 +348,34 @@ describe('Copy and paste', () => {
             });
             it('should paste a text when selection leave a span', async () => {
                 await testEditor(BasicEditor, {
-                    contentBefore: '<div>ab<span>c[d</span>e]f</div>',
+                    contentBefore: '<div>ab<span odoo-test-ignore-attr>c[d</span>e]f</div>',
```

Comment:
@dmo-odoo Is that a span we were supposed not to ignore for the relevance of the test?